backbone.js

```diff
@@ -76,6 +79,10 @@
   //
   // `Function#apply` can be slow so we use the method's arg count, if we know it.
   var addMethod = function(length, method, attribute) {
+    // In case we don't catch it, deopt Underscore functions that contain
+    // the arguments keyword.
+    if (length > 0 && containsArguments.test(_[method])) length = 0;
```

Comment:
No need for length>0 and I would create the regex on this line

`if (/\barguments\b/.test(_[method])) length = 0;`

-1 on this overall. I'd prefer we just be more careful with this in the future. Instead change this patch to set `reduce: 0, reduceRight: 0`